dex/fuzz/fuzz_targets/multiple_orders.rs

```diff
@@ -167,8 +170,10 @@ fn run_actions(actions: Vec<Action>) {
         solana_program::program_stubs::set_syscall_stubs(Box::new(NoSolLoggingStubs));
     }
 
+    // Toggle when to test a permissioned market.
+    let is_permissioned = actions.len() % 2 == 0;
```

Comment:
This seems like it's liable to interfere with shrinking. Instead, let's replace the `actions: Vec<Action>` argument with a `#[derive(Arbitrary)]` struct containing both setup parameters and the action sequence.

Reply:
Done.